tests/test_group.rs

```diff
@@ -150,109 +203,162 @@ fn basic_group_setup() {
     };
 }
 
-/*
 #[test]
+/// This test simulates various group operations like Add, Update, Remove in a small group
```

Comment:
To make this more digestible having a high level description of what's supposed to happen would be nice. Something like
* Alice creates group
* Alice adds Bob
* Bob removes Alice 
...

Reply:
Good point. I'll add this at the end in the last PR, because the order is still very much in flux. These tests need a better system anyway, to be able to cover more cases. For now the idea is to just re-enable the tests that were already written.

Reply:
Yeah, That's what @kkohbrok is currently working on (no PR yet).